internal/tray-agent/agent/client/agent-client.go

```diff
@@ -190,6 +194,8 @@ func GetAgentController() *AgentController {
 				if agentCtrl.ConnectionTest() {
 					if err = agentCtrl.StartCaches(); err == nil {
 						agentCtrl.connected = true
+						//init configuration data
+						agentCtrl.acquireClusterConfiguration()
```

Comment:
So, if I understand correctly, we are not able to change these configurations while the tray-agent is running, is it correct?

Reply:
This method only manages startup configuration the Agent uses to acquire config params (like the dashboard resources) and in future releases also its last configuration (e.g. level of notification). 
As soon as I Implement the update function for the ClusterConfig cache, we will be able to also have a runtime update.